pyproject.toml

```
@@ -99,6 +92,42 @@ bump = true
 enable = true
 style = "pep440"
 
+[tool.ruff]
+target-version = "py38"
+line-length = 127
+
+select = [
+    "F",     # Pyflakes
+    "W",     # Pycodestyle (warnigns)
+    "E",     # Pycodestyle (errors)
+    "N",     # pep8-naming
+    "ANN",   # flake8-annotations
+    "B",     # flake8-bugbear
+    "FA",    # flake8-future-annotations
+    "TID",   # flake8-tidy-imports
+    "UP007", # flake8-new-union-types equivalent rule
+    "RUF",   # ruff's custom rules
+]
+
+ignore = [
+    "ANN002", # Missing type annotation for *args
+    "ANN003", # Missing type annotation for **kwargs
+    "ANN101", # Missing type annotation for self in method
+    "ANN102", # Missing type annotation for cls in classmethod
+    "ANN204", # Missing return type annotation for special method
+    "B904",   # Exception raised within try-except should use raise ... from exc
+    "RUF012", # Mutable class attributes should be annotated with `typing.ClassVar`
+]
+
+[tool.ruff.extend-per-file-ignores]
+"test_*.py" = [
+    "ANN", # flake8-annotations
+]
+"docs/examples/code/**" = [
+    "FA",  # flake8-future-annotations
+]
+
```

Backport `ban-relative-imports=true` from `.flake8`.

```suggestion

[tool.ruff.flake8-tidy-imports]
ban-relative-imports = "all"
```

---

oh, good catch, I intended to do this but forgot after removing .flake8

---

Fixed in a [force push](https://github.com/py-mine/mcstatus/compare/75287f60b0ccbc8046eeec1b181972820df64b61..1c5cc62e09c2461608cd8c1572248a561cf80459)